ui/components/ConfirmationModal.js

```diff
@@ -234,24 +237,15 @@ function ConfirmationMsg(props) {
   const handleKubernetesClick = (ctxID) => {
     updateProgress({ showProgress : true })
     pingKubernetes(
-      successHandlerGenerator(enqueueSnackbar, closeButtonForSnackbarAction(closeSnackbar), "Kubernetes pinged", () => updateProgress({ showProgress : false })),
```

Comment:
have you changed the respective functions like successHandlerGenerator to accomodate the change